perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -485,6 +485,7 @@ sub direct_debit_modify : Private {
 
     my $ref = $c->stash->{orig_sub}->get_extra_metadata('payerReference');
     $p->set_extra_metadata(payerReference => $ref);
+    $p->confirm if $c->cobrand->moniker eq 'bexley';
```

Comment:
Calling `confirm` would mean the report gets sent to Whitespace - something we probably don't want to happen if we know there were DD errors. Looks like we only confirm new GGWs after APS was successful, so guess the same pattern should apply here?